docs/scos/dev/glue-api-guides/202311.0/glue-api-infrastructure/what-is-glue-application.md

```diff
@@ -0,0 +1,31 @@
+---
+title: What is API application?
+description: This document will describe what is API Application and why do we need it.
+last_updated: December 12, 2023
+template: glue-api-storefront-guide-template
+---
+
+Currently, Glue includes 3 applications: Legacy Glue API, Glue Storefront API and Glue Backend API. All those applications share a common infrastructure in [GlueApplication module](/docs/scos/dev/glue-api-guides/{{page.version}}/glue-api-infrastructure/glue-application-common-infrastrtructure.html), but also have a few infrastructural differences.
```

Comment:
"but also have a few infrastructural differences." same, same but different? What and why?

Reply:
can you provide a visualisation as and overview ?